### DISCUSSION TITLE
Object detection with YOLOv8 with pretrained model.

### DISCUSSION THREAD

**Discussion by Simon334** [ultralytics/ultralytics#9416] [Q&A]
Hi, I have a question, if I already have a trained model yolov8 and saved weights best.pt then how can I now check how the trained model will perform on another custom dataset without designated annotations

**Comment by pderrenger** [comment]
Hi! You can assess your trained model's performance on a new custom dataset without annotations by using the `predict` mode in YOLOv8. This will allow you to visualize the model's predictions on your images or video. If you're aiming for a qualitative assessment, this can be quite helpful. Here's an example on how to use the CLI for predictions:

```bash
yolo detect predict model=best.pt source=path/to/your/dataset/images/
```

Or if you prefer using Python, here's how you can do it:

```python
from ultralytics import YOLO

# Load your trained model
model = YOLO('best.pt')

# Make predictions on the dataset
results = model(['path/to/your/dataset/images/img1.jpg', 'path/to/your/dataset/images/img2.jpg'])

# Visualize the results
results.show()
```

Replace `'path/to/your/dataset/images/'` with the path to your custom dataset images or a single image. This way, you'll get the detection results that you can manually inspect. 😊

Hope this helps!
